### ISSUE TITLE
Failing Groovy Scripts

### ISSUE THREAD

**Issue by denniswisnia** [idealista/nexus-role#36]
### Prerequisites

* Nexus 3.21

### Description

On the newer Verrsions of Nexus, I got the following error:
```
TASK [aim-nexus : NEXUS | Calling Groovy script maven_create_proxy_repository] ***
fatal: [nexus-staging-02]: FAILED! => {"changed": false, "connection": "close", "content": "{\n  \"name\" : \"maven_create_proxy_repository\",\n  \"result\" : \"org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:\\nScript8.groovy: 12: You cannot create an instance from the abstract interface 'org.sonatype.nexus.repository.config.Configuration'.\\n @ line 12, column 17.\\n   configuration = new Configuration(\\n                   ^\\n\\n1 error\\n\"\n}", "content_length": "373", "content_type": "application/json", "date": "Mon, 09 Mar 2020 10:53:34 GMT", "elapsed": 0, "json": {"name": "maven_create_proxy_repository", "result": "org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:\nScript8.groovy: 12: You cannot create an instance from the abstract interface 'org.sonatype.nexus.repository.config.Configuration'.\n @ line 12, column 17.\n   configuration = new Configuration(\n                   ^\n\n1 error\n"}, "msg": "Status code was 400 and not [200, 204]: HTTP Error 400: Bad Request", "redirected": false, "server": "Nexus/3.21.0-05 (PRO)", "status": 400, "url": "http://localhost:8081/service/rest/v1/script/maven_create_proxy_repository/run", "x_content_type_options": "nosniff"}
```

### Steps to Reproduce

1. run ansible
2. try to create Maven Repositories
3. get the error.

**Expected behavior:** Create a Repositoriy or correct the settings

**Actual behavior:** Yea, it fails

**Comment by denniswisnia** [comment]
These Guys had the same issue, but I am not very familar with groovy scripts and dont understand how to fix it in the groovy scripts of this playbook:

https://github.com/ansible-ThoTeam/nexus3-oss/pull/247/commits/fc946c076adc3f6e39e958ce56e8796570a9bcf3#diff-b2b1ccc567b0b2cbb358989bdc5bf564

**Comment by jmonterrubio** [comment]
Thank you @denniswisnia, this is a known issue with the latests versions of nexus. We'll try to fix it asap! Meanwhile you can use the default version setted in the role.

**Comment by denniswisnia** [comment]
Yea, I would very happy about a quick fix. I think that this is a easy thing.
And we are trying right know to roll out the newest version. So I cant stay sadly on the the default version.

**Comment by vicsufer** [comment]
Hi there!
The error is fixed in [Release 2.0.0](https://github.com/idealista/nexus-role/releases/tag/2.0.0)
